setup.py

```diff
@@ -1,75 +1,27 @@
-from cx_Freeze import setup, Executable
-import os
-from os.path import dirname, join
-import requests
-import subprocess
-import sys
+#!/usr/bin/env python3
 
-from app.util import autoversioning
-from app.util.process_utils import is_windows
-
-
-buildOptions = {
-    'excludes': [],
-    'append_script_to_exe': False,
-    'build_exe': 'dist',
-    'compressed': True,
-    'copy_dependent_files': True,
-    'create_shared_zip': True,
-    'include_in_shared_zip': True,
-    'include_files': [
-        ('bin/git_askpass.sh', 'bin/git_askpass.sh'),
-        ('bin/git_ssh.sh', 'bin/git_ssh.sh'),
-        ('conf/default_clusterrunner.conf', 'conf/default_clusterrunner.conf'),
-        (requests.certs.where(), 'cacert.pem'),
```

Comment:
I assume all these files get included automatically?

Reply:
I need to investigate this. I was going to add them to the RPM packaging, but at least for default_clusterrunner.conf I can add it into the pex archive.

Reply:
Added the conf and scripts.
cacert.pem appears to be a workaround for cx_freeze, so it should no be longer needed. I remove the code that was referencing that file.

Reply:
Ah, yeah that sounds right. 👍